src/matchers/app.rs

```diff
@@ -32,6 +32,11 @@ pub fn is_exe(buf: &[u8]) -> bool {
     buf.len() > 1 && buf[0] == 0x4D && buf[1] == 0x5A
 }
 
+/// Returns whether a buffer is a DLL.
+pub fn is_dll(buf: &[u8]) -> bool {
+    is_exe(buf)
+}
+
```

Comment:
The fact that we can't distinguish between `dll` and `exe` should also be documented here.